database/migrations/2023_08_28_115131_update_training_tickets_add_monitor.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up() {
+        Schema::table('training_tickets', function ($table) {
+            $table->integer('monitor');
```

Comment:
Unless we plan to do different levels of monitoring at some point, I think we should use a Boolean here (it'll mask it as a bit instead of taking space for a full integer)

Reply:
Agreed - fixed and added a default value